### ISSUE TITLE
Search by gene name in pileup view

### ISSUE THREAD

**Issue by ihodes** [hammerlab/cycledash#763]
Use case: 

>>> KMT2D is mutated (at various points) in ~89% of follicular lymphoma. Can we interrogate this gene specifically (e.g. w less stringent filters)?

**Comment by danvk** [comment]
Can you say more about why you'd want to do this in the pileup view, as opposed to via CQL:

> annotations:gene_names = 'KMT2D'

**Comment by ihodes** [comment]
Definitely—the use-case is looking for evidence of a variant that *wasn't* called.
